[DO NOT MERGE] Global Settings instructions changes (#1649)
cloudflare/cloudflare-docs: products/cloudflare-one/src/content/identity/idp-integration/ping-saml.md

>     This file shares several fields with Cloudflare Access so you don’t have to input this data.

Reviewer: ```suggestion
    This file shares several fields with Cloudflare Access so you do not have to input this data.
```
